<launch>
  <node 
    pkg="livox_camera_calib"
    type="bag_to_pcd"
    name="bag_to_pcd"
    output="screen"
  />
   <param name="bag_file" type="string" value="/home/squirtle/Desktop/Squirtle/cv/ws_livox/data/basic1_and_advance4_calibration/livox_bag/1.bag"/>
   <param name="lidar_topic" type="string" value="/livox/lidar"/>
   <param name="pcd_file" type="string" value="/home/squirtle/Desktop/Squirtle/cv/ws_livox/data/basic1_and_advance4_calibration/my_pcd/1.pcd"/>
   <param name="is_custom_msg" type="bool" value="true"/>
</launch>